src/app/suggestion-notifications/qa/events/quality-assurance-events.component.html

```diff
@@ -0,0 +1,218 @@
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <h4 class="border-bottom pb-2">
+        {{'quality-assurance.events.topic' | translate}} {{this.showTopic}}
+      </h4>
+
+      <ds-loading class="container" *ngIf="(isEventPageLoading | async)" message="{{'quality-assurance.loading' | translate}}"></ds-loading>
+
+      <ds-pagination *ngIf="!(isEventPageLoading | async)"
+                      [paginationOptions]="paginationConfig"
+                      [collectionSize]="(totalElements$ | async)"
+                      [sortOptions]="paginationSortConfig"
+                      (paginationChange)="getQualityAssuranceEvents()">
+
+        <ng-container>
+          <div *ngIf="(eventsUpdated$|async)?.length == 0" class="alert alert-info w-100 mb-2 mt-2" role="alert">
+            {{'quality-assurance.noEvents' | translate}}
+          </div>
+          <div *ngIf="(eventsUpdated$|async)?.length != 0" class="table-responsive mt-2">
+            <table id="events" class="table table-striped table-hover table-bordered">
+              <thead>
+              <tr>
+                <th scope="col" class="trust-col">{{'quality-assurance.event.table.trust' | translate}}</th>
+                <th scope="col" class="title-col">{{'quality-assurance.event.table.publication' | translate}}</th>
+                <th *ngIf="hasDetailColumn() && showTopic.indexOf('/PROJECT') == -1" scope="col" class="content-col">
+                  {{'quality-assurance.event.table.details' | translate}}
+                </th>
+                <th *ngIf="hasDetailColumn() && showTopic.indexOf('/PROJECT') !== -1" scope="col" class="content-col">
+                  {{'quality-assurance.event.table.project-details' | translate}}
+                </th>
+                <th scope="col" class="button-col">{{'quality-assurance.event.table.actions' | translate}}</th>
+              </tr>
+              </thead>
+              <tbody>
+                <tr *ngFor="let eventElement of (eventsUpdated$ | async); let i = index">
+                  <td>{{eventElement?.event?.trust}}
+                  </td>
+                  <td><a *ngIf="eventElement?.target"
+                         target="_blank"
+                         [routerLink]="['/items', eventElement?.target?.id]">{{eventElement.title}}</a>
+                    <span *ngIf="!eventElement?.target">{{eventElement.title}}</span>
+                  </td>
+                  <td *ngIf="showTopic.indexOf('/PID') !== -1">
+                    <p><span class="small">{{'quality-assurance.event.table.pidtype' | translate}}</span>&nbsp;<span class="badge badge-info">{{eventElement.event.message.type}}</span></p>
+                    <p><span class="small">{{'quality-assurance.event.table.pidvalue' | translate}}</span><br>
+                      <a *ngIf="hasPIDHref(eventElement.event.message); else noPID" href="{{getPIDHref(eventElement.event.message)}}" target="_blank">
```

Comment:
Per the CodeQL alert on this line of code, this `<a>` tag should likely have `rel="noopener noreferrer"` added as attributes, since it has `target="_blank"`.